lambda_functions/app.py

```diff
@@ -215,7 +215,24 @@ def create_user(event, context):
             "body": "Could not create user. Check HTTP POST payload. Exception: {}".format(e)
         }
 
-
+def create_user_from_cognito(event, context):
+    try:        
+        user = User()
+        user.name = event.UserName
```

Comment:
Is this a special kind of event? Because usually Lambda events do not contain a UserName or request/userAttributes/sub field

Reply:
Yes, the incoming event should look like described here:
https://stackoverflow.com/questions/49580078/send-username-to-aws-lambda-function-triggered-by-aws-cognito-user-confirm